build.sbt

```diff
@@ -25,7 +25,7 @@ val zioInteropCats2Version    = "22.0.0.0"
 val zioInteropCats3Version    = "23.0.0.1"
 val zioInteropReactiveVersion = "2.0.0"
 val zioConfigVersion          = "3.0.7"
-val zqueryVersion             = "0.3.6"
+val zqueryVersion             = "0.3.6+1-45b3ae66-SNAPSHOT"
```

Comment:
I think there was a release so you can use the official one now 🎉

Reply:
Seems like it failed to get published, I'll ping Adam on discord!